docs/cow-protocol/tutorials/arbitrate/orderbook.md

```diff
@@ -0,0 +1,120 @@
+
+The orderbook may simulate quotes if possible to make sure their proposed solution would pass given the current state of the chain and picks the one maximizing the traders' output (the solver winning the quote is rewarded by the protocol). 
+The quote is turned into an order that is returned to the Trader and may be signed.
+
+Once signed, the trader will post the order for inclusion in the next batch.
+At this stage the orderbook will verify the order is valid and either fetch the existing quote or create a new one (the "expected" out amount at order placement time is relevant for order classification and fee policies).
+Finally it stores the order in the database, so that the autopilot can index it, and returns the uniquer identifier to the trader.
+
+Now it's the protocols and solvers' turn to distribute and match the order.
+During this time, the trader may request status updates about their order from the orderbook and finally request information about the executed trade and solver competition.
+
+## Methodology (how the component works)
+
+The orderbook exposes a variety of different endpoints 
+The majority of them are simple REST interfaces on top of the data model that is stored in the database.
+However, there are some more involved operations, which are worth explaining in more detail:
+
+### Getting quotes
+
+The quoting process piggy-backs on the solver competition.
+Solver are tasked in finding a matching for an auction instance including only one order (the order to be quoted).
+These quote auctions have a significantly shorter deadline to solve "quote" auctions as they are fundamentally simpler to solve.
+
+Quotes can be requested using either the "fast" or "optimal" quality type.
+
+Optimal quotes wait for all connected solvers to either return a response or time out before yielding a result.
+
+Fast quotes return the best estimate as soon as a threshold of solvers (ie. two) have returned a successful estimate.
+
+The API also exposes a `/native_price` route which can be used to get even more indicative estimates (ie for showing approximate dollar or ETH amounts of a trade).
+These price estimates use fixed amounts and are more heavily cached allowing for faster response times.
+
+### Placing orders
+
+Before an order is accepted in the database it is validated.
+Validation steps include
+- Asserting the order is well formed and all fields are properly encoded
+- The user has sufficient balance/allowance to place this order
+- The signature types is supported and valid
+- The pre-image for the order's appData hash is either provided in the request or publicly available on IPFS (to ensure any additional information about the order can be properly interpreted by the orderbook)
+
+Each order is also associated with a quote for classification into in or out of market price (which may have an effect on the fee policy used).
```

Comment:
Phrasing nit for readability, feel free to ignore.

```suggestion
Each order is also associated with a quote for classification as either in or out of market price. This may have an effect on the fee policy used.
```